include/_kernel_c.h

```diff
@@ -223,14 +223,6 @@ typedef struct _pocl_image2d_array_t { dev_image_t base; }* image2d_array_t;
 typedef struct _pocl_image1d_array_t { dev_image_t base; }* image1d_array_t;
 #endif
 
-// 3.9 needs access qualifier
-// TODO: rw images
-#ifdef CLANG_OLDER_THAN_3_9
```

Comment:
Nope, it just removes AS qualifiers (ro/wo/rw) on images for older clangs (since they ignore them, and it would lead to multiple identical prototypes). Commit a54ff9a88bf85a52008 (unrelated to this) makes 3.7 + HSA compile again (on my HSA buildbot).